### DISCUSSION TITLE
Victron in HA

### DISCUSSION THREAD

**Discussion by viniciogrigoli** [indomus/forum#2896] [HUB: Home Assistant]
Buongiorno a tutti, ho installato su un Raspberry Pi4 Home Assistant e lo utilizzo sul camper e dato che ho dei componenti Victron shunt, MPPT, DC/DC e Caricabatterie che possono essere gestite dall'app della Victron tramite bluethoot, volevo sapere se è possibile integrarli in HA per un controllo da remoto.
Grazie

**Comment by indomus** [comment]
https://indomus.it/formazione/home-assistant-come-capire-come-integrare-un-componente-domotico-o-meno/
